### PR TITLE
instr(metrics): Ignore invalid timestamp errors

### PR DESCRIPTION
**Pull request by jjbayer**
Ignore `InvalidTimestamp` error on metrics aggregation.

The metrics aggregator only accept timestamps from the last 5 days. But incoming metrics buckets can always contain invalid timestamps, even when we validated the timestamps on the payload before, because the timestamp may have become invalid while the metric was aggregated in a downstream relay.

With custom metrics, there is no pre-validation at all, so we will see even more `InvalidTimestamp` errors as their usage increases.

ref: https://github.com/getsentry/team-ingest/issues/227

#skip-changelog

### CODE REVIEW

File: relay-metrics/src/aggregatorservice.rs [hunk 1]
```diff
@@ -265,16 +264,8 @@ impl AggregatorService {
...
-                .merge_all(project_key, buckets, self.max_total_bucket_bytes)
-        {
-            relay_log::error!(
-                tags.aggregator = &self.aggregator.name(),
-                error = &err as &dyn Error,
-                "failed to merge buckets"
-            );
-        }
+        self.aggregator
+            .merge_all(project_key, buckets, self.max_total_bucket_bytes);
```
Comment (jjbayer) [inline relay-metrics/src/aggregatorservice.rs:268]: Drive-by refactor: `merge_all` always returned `Ok(())`, so I made it return `()` instead.

**Review by olksdr — APPROVED** [review]
